lib/ODataClient.d.ts

```diff
@@ -5,6 +5,10 @@ export declare class ODataClient {
     defaultRequestConfig: AxiosRequestConfig;
     private httpClient;
     constructor(baseUrl: string, config?: AxiosRequestConfig);
+    readonly interceptors: {
+        request: import("axios").AxiosInterceptorManager<AxiosRequestConfig>;
```

Comment:
Would rather have the types imported at the top of the file instead of inline imports.